app/models/filters/filter_base.rb

```diff
@@ -283,14 +289,16 @@ def known_params
         prior_living_situation_ids: [],
         length_of_times: [],
         times_homeless_in_last_three_years: [],
+        secondary_project_ids: [],
+        secondary_project_group_ids: [],
       ]
     end
 
     def all_known_keys
       known_params.map { |k| if k.is_a?(Hash) then k.keys else k end }.flatten
     end
 
-    def selected_params_for_display(single_date: false)
+    def selected_params_for_display(single_date: false, secondary_projects_label: 'Secondary Projects', secondary_project_groups_label: 'Secondary Project Groups')
```

Comment:
maybe this should be:
```suggestion
    def selected_params_for_display(single_date: false, labels: {})
```
Then we could pass in any labels we wanted, just check `labels[:key] || 'default label'`